packages/core/lib/console.js

```diff
@@ -16,13 +16,13 @@ const { spawn } = require("child_process");
 const Require = require("@truffle/require");
 const debug = require("debug")("console");
 const { getCommand, parseQuotesAndEscapes } = require("./command-utils");
-const validTruffleCommands = require("./commands/commands");
+const { validTruffleConsoleCommands } = require("./commands/commands");
 
 // Create an expression that returns a string when evaluated
 // by the REPL
 const makeIIFE = str => `(() => "${str}")()`;
 
-const processInput = (input, allowedCommands) => {
+const processInput = input => {
```

Comment:
While working this PR I was tempted to refactor REPL input processing because it is all over the place, misnamed and should be consolidated. I'll add an issue for that.